packages/common/src/types.ts

```diff
@@ -386,9 +386,9 @@ export enum ProviderRpcErrorCode {
 export interface Chain {
   namespace?: 'evm'
   id: ChainId
-  rpcUrl: string
-  label: string
-  token: TokenSymbol // eg ETH, BNB, MATIC
+  rpcUrl?: string
```

Comment:
It would be helpful to add the notes for these props (and any missing) found in the README to the types as ts notes so they show up on hover in the IDE
EX:
```suggestion
  /**
   * Recommended to include. Used for network requests (eg Alchemy or Infura end point).
   * PLEASE NOTE: Some wallets require an rpcUrl, label, 
   * and token for actions such as adding a new chain. 
   * It is recommended to include rpcUrl, label, and token for full functionality. 
   */
  rpcUrl?: string
```
...or whatever fits within the prettier line-length max